common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1016,14 +1021,14 @@ private AuthenticationConstants() {
         public static final String BROWSER_EXT_INSTALL_PREFIX = "msauth://";
 
         /**
-         * Prefix in the redirect to open external browser to finish the CA auth.
+         * A query param indicating that this is an intune device CA link.
          */
-        public static final String BROWSER_DEVICE_CA_URL = "browser://go.microsoft.com/fwlink/?LinkId=396941";
+        public static final String BROWSER_DEVICE_CA_URL_QUERY_STRING_PARAMETER = "&ismdmurl=1";
```

Comment:
Looks we already had the constant defined using the forward link instead of the query string and were using it to launch CP. So server change was not a must but good to have.

Reply:
I actually prefer not hardcoding this - as the link defined by Intune could be changed on the server side at any point in time.